task/buildah-remote/0.1/buildah-remote.yaml

```diff
@@ -234,6 +241,10 @@ spec:
         BUILDAH_ARGS+=("--target=${TARGET_STAGE}")
       fi
 
+      if [ -n "${BUILD_ARGS_FILE}" ]; then
+        BUILDAH_ARGS+=("--build-arg-file=${SOURCE_CODE_DIR}/${CONTEXT}/${BUILD_ARGS_FILE}")
```

Comment:
How will precedence be determined between the buildah args that we set and those that are set via this file? Should we do any inspection of the args file to warn/fail if there is collision between the arguments?

Reply:
these are use as --build-arg argument=value   to be exposed as env variables during build

those aren't regular arguments for buildah build

and since we don't pass any --build-arg, there isn't any conflict